internal/handlers/pg/pgdb/query.go

```diff
@@ -64,6 +65,8 @@ type SQLParam struct {
 // Context cancellation is not considered an error.
 //
 // If the collection doesn't exist, fetch returns a closed channel and no error.
+//
+// Deprecated in favor of GetDocuments, TODO remove in https://github.com/FerretDB/FerretDB/issues/898.
```

Comment:
It should be `Deprecated:`, with a colon

https://github.com/golang/go/wiki/Deprecated

Reply:
fixed